pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools~=63.0", "wheel"]
+requires = ["setuptools~=66.1", "wheel"]
```

Comment:
@dgw I made the minimal adjustment to this pin from the value set by 557b44c, but it seems prudent to ask if we have any interest in relaxing the pin now. I don't really know what the original issue was, but I can perform an `--editable` install with both 3.9 and 3.12 with the pin as I have it here.